src/BridgeGameManager.js

```diff
@@ -0,0 +1,107 @@
+const BridgeGame = require('./Model/BridgeGame');
+const BridgeMaker = require('./BridgeMaker');
+const InputView = require('./View/InputView');
+const OutputView = require('./View/OutputView');
+const Validation = require('./libs/Validation');
+const { Console } = require('@woowacourse/mission-utils');
+const { COMMAND_OPTION, MESSAGE } = require('./libs/Constant');
+const { generate } = require('./BridgeRandomNumberGenerator');
+const { throwException } = require('./libs/ErrorHandler');
+
+class BridgeGameManager {
+  #bridgeGame;
+  #isRetry;
+
+  constructor() {
+    this.#bridgeGame = null;
+    this.#isRetry = false;
+  }
```

Comment:
```suggestion
  #bridgeGame = null;;
  #isRetry = false;
```
저는 `constructor`에서 프로퍼티에 값을 초기화 할 때 인자로 받아서 값을 초기화 해줘야 할 경우에만 `constructor` 안에서 처리하는 것이 가독성이 더 좋다고 판단하였습니다!

Reply:
좋은 의견 감사합니다!
확실히 위에서 `constructor`은 없어도 충분할 거 같아요ㅎㅎ 리팩터링을 진행하면서 고쳐보겠습니다!